cea/datamanagement/create_new_project.py

```diff
@@ -26,37 +28,39 @@
 COLUMNS_ZONE_GEOMETRY = ['Name', 'floors_bg', 'floors_ag', 'height_bg', 'height_ag']
```

Comment:
let's keep in mind that the end goal should be to have this defined somewhere centrally for the whole CEA... i think the glossary efforts could end up producing something like this, maybe a YAML file with information about input and output formats?